src/app/components/device-list/deviceList.css

```diff
@@ -0,0 +1,18 @@
+th {
```

Comment:
<th>Type</th>
<th>Description</th>

they are not clickable, but have cursor pointer

use classes instead